usaspending_api/download/management/commands/populate_monthly_delta_files.py

```diff
@@ -321,61 +335,6 @@ def check_regex_match(self, award_type, file_name, generate_since):
 
         return "{}-{}-{}".format(year, month, day)
 
-    @staticmethod
-    def apply_annotations_to_sql(raw_query, aliases):
```

Comment:
`populate_monthly_delta_files.apply_annotations_to_sql()` was slightly different than the original `download_generation.apply_annotations_to_sql()`.

It doesn't appear the new `apply_annotations_to_sql()` function was updated when it replaced this function. Have manual regression tests been run for delta files?

Reply:
The difference was to account for UNION statements within the sql. The usage in `populate_monthly_delta_files.py` accounts for that. Running the command was smoke-tested.

Reply:
Cool, just checking.